crypto/fipsmodule/ec/ecp_nistz256_table.c

```diff
@@ -0,0 +1,9542 @@
+/*
```

Comment:
This table is already in *ring* and we won't need this file.

The PPC select functions are using the scatter/gather technique, which we don't need in *ring*. It will be better to just remove the "scatter-gather subroutines" section of the ecp_nistz256-ppc64.pl file and then remove ecp_nistz256_table.c too.